core/common/src/main/java/alluxio/underfs/UnderFileSystemWithLogging.java

```diff
@@ -1203,14 +1207,27 @@ public String toString() {
    */
   private <T> T call(UfsCallable<T> callable) throws IOException {
     String methodName = callable.methodName();
+    long startMs = System.currentTimeMillis();
+    long durationMs;
     LOG.debug("Enter: {}: {}", methodName, callable);
     try (Timer.Context ctx = MetricsSystem.timer(getQualifiedMetricName(methodName)).time()) {
       T ret = callable.call();
-      LOG.debug("Exit (OK): {}: {}", methodName, callable);
+      durationMs = System.currentTimeMillis() - startMs;
+      LOG.debug("Exit (OK): {}({}) in {} ms", methodName, callable, durationMs);
+      if (durationMs >= THRESHOLD) {
+        LOG.warn("{}({}) returned OK in {} ms (>={} ms)", methodName,
+            callable, durationMs, THRESHOLD);
+      }
       return ret;
     } catch (IOException e) {
+      durationMs = System.currentTimeMillis() - startMs;
       MetricsSystem.counter(getQualifiedFailureMetricName(methodName)).inc();
-      LOG.debug("Exit (Error): {}: {}, Error={}", methodName, callable, e.getMessage());
+      LOG.debug("Exit (Error): {}({}) in {} ms, Error={}",
+          methodName, callable, e.getMessage(), durationMs);
```

Comment:
I think the arguments are in the wrong order. `e.getMessage()` and `durationMs` should be switched.